diagnostic.md

```diff
@@ -184,5 +210,6 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
+70px
```

Comment:
At 550px screen width, the `@media(min-width=400px)` would take place and `.stretch` is 25%. So 25% * 550px = 137.5px